versioned_docs/version-23.04/graph-views/map-web-update.md

```diff
@@ -32,9 +32,37 @@ sudo dnf update centreon-map-engine centreon-map-web-client
 </TabItem>
 <TabItem value="Debian 11" label="Debian 11">
 
-``` shell
-sudo apt update centreon-map-engine centreon-map-web-client
-```
+ - If MAP only is installed:
+ 
+ ``` shell
+ sudo apt upgrade centreon-map-engine centreon-map-web-client
+ ```
+  
+ - If MAP and MAP Legacy are installed on the same server:
+   
+   - Make a backup of the **map.cnf** file:
+    
+    ```shell
+    cp /etc/mysql/map.cnf /etc/mysql/map.cnf.bk
```

Comment:
The correct path for map.cnf on Debian is `/etc/my.cnf.d/map.cnf`
so the backup copy should be `cp /etc/my.cnf.d/map.cnf /etc/my.cnf.d/map.cnf.bk`